lib/brakeman/checks/check_forgery_setting.rb

```diff
@@ -52,6 +52,17 @@ def run_check
         :confidence => CONFIDENCE[:high],
         :gem_info => gemfile_or_environment,
         :link_path => "https://groups.google.com/d/topic/rubyonrails-security/LZWjzCPgNmU/discussion"
+    elsif version_between? "4.0.0", "100.0.0" and forgery_opts = app_controller[:options][:protect_from_forgery]
+
+      unless forgery_opts.is_a?(Array) and sexp?(forgery_opts.first) and hash_access(forgery_opts.first.first_arg, :with).value == :exception
+        warn :controller => :ApplicationController,
```

Comment:
done 12ed187aaa85ab560c927bc8585f75e3a6efefbf. sometimes the value is a boolean so there's some code to handle that.